apps/web-portal/lib/WebPortalExpressApp.mjs

```diff
@@ -0,0 +1,45 @@
+import assert from 'assert'
+import express from 'express'
+import * as RestApiV1 from './rest-api-v1'
+import {makeRunner} from '@zorko-io/util-use-case'
+
+// TODO: gh-80 handle process start/stop, wire with logger, etc
+export class WebPortalExpressApp {
+  /**
+   * @type {Express}
+   */
+
+  #http = null
+  #config = null
+
+  /**
+   * @param {Object} context
+   * @param {Object} context.config - app config
+   */
+
+  constructor(context = {}) {
+    assert(context.config, 'Should have an app config defined')
+
+    this.#config = context.config
+    this.#http = express()
+
```

Comment:
not see app middlewares(json, cors, urlencode, etc), maybe add todo?

Reply:
added todo